checkov/github/checks/disallow_inactive_branch.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+import datetime
+from typing import Any
+
+from jsonpath_ng import parse
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.github.base_github_configuration_check import BaseGithubCheck
+from checkov.github.schemas.branch import schema as branch_schema
+from checkov.json_doc.enums import BlockType
+
+
+class GithubDisallowInactiveBranch(BaseGithubCheck):
+    def __init__(self) -> None:
+        name = "Ensure inactive branches are reviewed and removed periodically - CIS 1.1.8"
+        id = "CKV_GITHUB_15"
+        categories = (CheckCategories.SUPPLY_CHAIN,)
+        super().__init__(
+            id=id,
+            name=name,
+            categories=categories,
+            supported_entities=("*",),
+            block_type=BlockType.DOCUMENT,
+        )
+
+    def scan_entity_conf(self, conf: dict[str, Any], entity_type: str) -> CheckResult | None:  # type:ignore[override]
+        if branch_schema.validate(conf):
+            jsonpath_expression = parse("$..{}".format(self.get_evaluated_keys()[0].replace("/", ".")))
```

Comment:
maybe move `self.get_evaluated_keys()[0].replace("/", ".")` to a variable and then leverage f-string literal instead of `format()`